### PR TITLE
small updates and fixes for Spanish translation

### PR DESCRIPTION
**Pull request by auroszx**
This is just a small update translating the new strings and fixing some errors I didn't notice before. Please let me know if anything should be fixed. Thank you.

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -2,6 +2,12 @@
 <resources>
     <string name="app_name">wallabag</string>
     <string name="label_name">¡Guárdalo!</string>
+<!--
```
Comment (tcitworld) [inline app/src/main/res/values-es/strings.xml]: Please remove this.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -211,8 +222,10 @@
 
     <string name="connectionWizard_providerSelection_titleMessage">Selecciona un proveedor de wallabag</string>
     <string name="connectionWizard_providerSelection_provider_general_name">Otro</string>
-    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir detalles manualmente</string>
+    <string name="connectionWizard_providerSelection_provider_v2wallabagorg_name" translatable="false">v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_general_desc">Selecciona esta opción para introducir los ajustes requeridos manualmente</string>
     <string name="connectionWizard_providerSelection_provider_v2wallabagorg_desc">Selecciona esta opción si tienes una cuenta en https://v2.wallabag.org</string>
+    <string name="connectionWizard_providerSelection_provider_framabag_name" translatable="false">Framabag.org</string>
```
Comment (Strubbl) [inline app/src/main/res/values-es/strings.xml]: Does anybody know if we really need the strings which are `translatable="false"` in every language file? I would remove them, too. But i do not know if this has any impact if those strings are only available in the values/strings.xml

Comment (di72nn) [inline app/src/main/res/values-es/strings.xml]: `translatable="false"` strings should be present only in default locale files (in `res/values/` directory).

**Comment by Strubbl** [comment]
Thank you very much for your contribution.

**Comment by auroszx** [comment]
No problem. I'll do what I can to keep the translation up to date.

**Pull request merged**